src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/reports/claimsbyhearingvenue/ClaimsByHearingVenueReportParams.java

```diff
@@ -0,0 +1,19 @@
+package uk.gov.hmcts.ethos.replacement.docmosis.reports.claimsbyhearingvenue;
+
+import lombok.EqualsAndHashCode;
+import lombok.Value;
+import uk.gov.hmcts.ethos.replacement.docmosis.reports.ReportParams;
+
+@Value
+@EqualsAndHashCode(callSuper = false)
+public class ClaimsByHearingVenueReportParams extends ReportParams {
+    String hearingType;
+    String currentUserFullName;
+
+    public ClaimsByHearingVenueReportParams(String caseTypeId, String dateFrom, String dateTo,
+                                            String hearingDateType,  String userFullName) {
+        super(caseTypeId, dateFrom, dateTo);
+        hearingType = hearingDateType;
```

Comment:
the convention in every other class is to write
this.hearingDateType = hearingDateType
this.userFullName = userFullName